src/Compiler/Compiler.cpp

```diff
@@ -70,22 +72,29 @@ extern const char* foedag_version_number;
 extern const char* foedag_git_hash;
 extern const char* foedag_build_type;
 
-auto CreateDummyLog = [](ProjectManager* projManager,
-                         const std::string& outfileName) {
+auto CreateDummyLog =
+    [](ProjectManager* projManager,
+       const std::string& outfileName) -> std::filesystem::path {
+  std::filesystem::path outputPath{};
+
   if (projManager) {
     std::filesystem::path projectPath(projManager->projectPath());
-    std::filesystem::path outputPath = projectPath / outfileName;
-    std::filesystem::remove(outputPath);
+    outputPath = projectPath / outfileName;
+    if (FileUtils::FileExists(outputPath)) {
+      std::filesystem::remove(outputPath);
+    }
     std::ofstream ofs(outputPath);
     ofs << "Dummy log for " << outfileName << "\n";
```

Comment:
This can be removed now since we have header in each rpt file.

Reply:
Without this text the dummy logs are completely empty and it would be hard to verify that the original log contents is included when adding the log header.

Reply:
Got it.